2-projects/1-todo-list/script.js

```diff
@@ -17,9 +41,58 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let input = document.querySelector(".form-control");
+  let inputValue = [
+    {
+      task: input.value,
+      completed: false,
+    },
```

Comment:
really good that you're replicating the same format as the objects in the todo list array here :)